### ISSUE TITLE
a11y: Review Menu Accessibility

### ISSUE THREAD

**Issue by StephenRiosDev** [AlaskaAirlines/auro-formkit#456]
### Description

Assess accessibility of the Menu component. It should work similarly to a radio group for screen-reader purposes. Ensure it meets all W3C standards and that the screen reader functions as expected.

W3C radio group example: https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio/

Note: It's ok if the selection doesn't change until the user presses a key to interact, but the controls should be explained to the user if they differ from a standard radio group.

Criteria:
- Only label should be read (currently all contents including the icon are read)
- Label should be read when the user hovers or hi-lights with arrow keys
- Selected state should be read when the item is hi-lighted
- Label and selected state should be read when the item is selected

### Exit criteria (To be determined by the Auro team)

Menu component should be accessible to non-sighted users and meet or exceed all related W3C WCAG 2.0 AA criteria.

**Comment by blackfalcon** [comment]
The floor-wide standard is to meet or exceed WCAG 2.2 Level AA. Please see [Alaska Airlines Digital Accessibility and Inclusion Policy](https://wiki.devtools.teamaag.com/standards/accessibility/digital-accessibility-policy) for more information.
